pkg/condition/condition.go

```diff
@@ -20,6 +20,15 @@ func AddOrUpdateStatusConditions(conditions []toolchainv1alpha1.Condition, newCo
 	return conditions, atLeastOneUpdated
 }
 
+func FindConditionByType(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) toolchainv1alpha1.Condition {
```

Comment:
It's not clear what this function returns if no condition found. You will have to look at the function implementation to figure it out.
Let's modify it to
```
func FindConditionByType(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) (toolchainv1alpha1.Condition, bool)
```
where the returned bool is true if found.
OR
```
func FindConditionByType(conditions []toolchainv1alpha1.Condition, conditionType toolchainv1alpha1.ConditionType) *toolchainv1alpha1.Condition
```
And also please add a description (and a test as Xavier mentioned).

Reply:
Done https://github.com/codeready-toolchain/toolchain-common/pull/36/commits/a247e08b1bed03a3437043ae547c370912ad3eaa